model/src/main/proto/arguments.proto

```diff
@@ -63,3 +64,31 @@ message InputInteractionViewTestActivityParams {
     MATH_EQUATION = 3;
   }
 }
+
+// Corresponds to intent extras of destination activity.
```

Comment:
This needs more detail, I think, such as an explanation for how this is used for routing.

The oneof and the recently_plahyed_activity_params should also be documented (see other oneofs for reference on what level of detail is added for each part of the proto).

Reply:
Improved KDoc